WordPress/WordPressTest/PeopleRemoteTests.swift

```diff
@@ -14,49 +21,29 @@ class PeopleServiceTests: XCTestCase {
     let deleteFollowerSuccessMockFilename   = "people-delete-follower-success.json"
     let deleteViewerFailureMockFilename     = "people-delete-viewer-failure.json"
     let deleteViewerSuccessMockFilename     = "people-delete-viewer-success.json"
-    let contentTypeJson                     = "application/json"
-    let timeout                             = TimeInterval(1000)
 
     // MARK: - Properties
 
-    var restApi: WordPressComRestApi!
     var remote: PeopleRemote!
```

Comment:
I know it's not part of this PR but could we rename the subject class (and related classes) to `PeopleServiceRemote` to be consistent?